frontends/web/src/locales/en/app.json

```diff
@@ -68,13 +68,11 @@
       "step": "Finished",
       "title": "Account added"
     },
-    "title": "Add account"
+    "title": "+ Add account"
```

Comment:
this seems wrong, is this correct @thisconnect? does not seem like a `+` should be part of i18n/translations.

Reply:
Yeah this is new and was added on purpose, it needs to be part of the translation as  right-to-left langs should put it on the other side.

Reply:
@thisconnect It's meant to be an SVG, will text "+" look good? :)

Reply:
@jadzeidan I wasn't aware of that. It looks as good as in your mockups.